<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pamietnik</class>
 <widget class="QMainWindow" name="Pamietnik">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pamietnik</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateTimeEdit" name="dataWpisuDateTime">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>510</y>
      <width>194</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addWpisButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>500</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj Wpis</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="listaWpisowTextEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>741</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>741</width>
      <height>291</height>
     </size>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="trescWpisuTextEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>741</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="previousWpisTextEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>741</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="currentWpisTextEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>741</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="nextWpisTextEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>741</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="previousWpisPushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Poprzednie</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextWpispPushButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>360</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Nastepne</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeWpisPushButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>360</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Usu≈Ñ Aktualny Wpis</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
